src/elements/FlexRow/FlexRow.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { getBreakpoint, numberLikeProp } from '../../util/helpers';
+import Common from '../Common/Common';
+
+const FlexRow = ({ children, breakMedium, breakSmall, breakCustom, gutter = 'none', ...other }) => {
+	const combinedClasses = classNames(
+			'ui-flex-row',
+			`gutter-${gutter}`,
+			other.classes
+		),
+		breakpoint = getBreakpoint(),
+		style = {};
+
+	switch (breakpoint) {
+		case 'Medium':
+			if(breakMedium) style.display = 'block';
+			break;
+
+		case 'Small':
+			if(breakMedium || breakSmall) style.display = 'block';
+			break;
+	}
+
+	if(breakCustom && window.innerWidth <= Number(breakCustom)) style.display = 'block';
+
+	/**
+	 * Render the grid items (`renderGridItems` defined below)
```

Comment:
Same thing about the `renderGridItems` part of the comment

Reply:
Sometimes i'm completely blind to comments